ext/opentelemetry-ext-django/src/opentelemetry/ext/django/middleware.py

```diff
@@ -32,6 +34,21 @@
 _logger = getLogger(__name__)
 
 
+def _disable_trace(url):
+    excluded_hosts = configuration.Configuration().DJANGO_EXCLUDED_HOSTS
+    excluded_paths = configuration.Configuration().DJANGO_EXCLUDED_PATHS
+
+    if excluded_hosts:
+        excluded_hosts = str.split(excluded_hosts, ",")
```

Comment:
this could be quite expensive if used over and over again. is there the ability to memoize this somehow?

This relies on the implementation of Configuration, but since it's read-only except for testing purposes, you should be able to memoize once and not use it any more. 

I could imagine excluded_hosts being a really long string, and paying the O(n) cost splitting every time can be expensive.